x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/transport/SecurityServerTransportInterceptor.java

```diff
@@ -301,7 +301,7 @@ private <T extends TransportResponse> void sendWithCrossClusterAccessHeaders(
                     );
                 }
 
-                logger.debug(
+                logger.trace(
```

Comment:
I think it's too verbose to DEBUG log every single request coming through.

Reply:
Good call. I'd also change this logging to use `Supplier` argument as well to avoid string concatenation on a hot path.